jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1596,24 +1596,30 @@ def reducer(arg1: TfVal, arg2: TfVal) -> TfVal:
       base_dilation, window_dilation, _in_avals, _out_aval
   )
 
-# _try_tf_max_pool returns a Tensor when it succeeds, or a string describing why
-# it did not succeed otherwise.
-def _try_tf_max_pool(operand, window_dimensions, window_strides, padding,
-                     base_dilation, window_dilation) -> Union[str, TfVal]:
-  # TODO(bchetioui): this function is not exhaustive wrt which
-  # reduce_window_max cases can be translated into a call to max_pool. Further
-  # investigation is needed to fully flesh it out.
 
+# _try_tf_pool returns a Tensor when it succeeds, or a string describing why
+# it did not succeed otherwise. It currently only supports reduce_window_max
+# and reduce_window_sum.
+# TODO(bchetioui): this function is not exhaustive wrt which
+# reduce_window_max or reduce_window_sum cases can be translated into a call to
+# max_pool or avg_pool. Further investigation is needed to fully flesh it out.
+def _try_tf_pool(op_name, operand, window_dimensions, window_strides, padding,
+                 base_dilation, window_dilation) -> Union[str, TfVal]:
   # Contrarily to the main path, tf.int8 is actually a valid type for
   # tf.nn.max_pool.
-  if operand.dtype in [tf.bool, tf.uint32, tf.uint64, tf.complex64,
-                       tf.complex128]:
+  if op_name == "reduce_window_max" and operand.dtype in [
+      tf.bool, tf.uint32, tf.uint64, tf.complex64, tf.complex128
+  ]:
     return f"tf.nn.max_pool does not support operands of type {operand.dtype}"
+  if op_name == "reduce_window_sum" and operand.dtype not in [
+      tf.half, tf.float16, tf.float32, tf.float64
```

Comment:
I don't think there is a difference between `tf.half` and `tf.float16`:
```
>>> tf.half is tf.float16
True
```
I think we could just use `tf.float16`? :)

Reply:
I would also prefer tf.float16; I do not think we have any uses of tf.half in the code base.

Reply:
Yep, I missed this. Done.